js/Inspector.js

```diff
@@ -112,11 +114,10 @@ export default function Inspector({ className, ...props }) {
       {paneComponent}
 
       <footer className='theme-dark'>
-        NPMGraph v${ENV.appVersion} ${'\xa9'} Robert Kieffer, 2020  MIT License – <ExternalLink
-            id='github'
-            href='https://github.com/npmgraph/npmgraph'>
-          GitHub
-        </ExternalLink>
+        NPMGraph v{ENV.appVersion} {'\xa9'}, MIT License
+        (See <ExternalLink id='github' href='//github.com/npmgraph/npmgraph'>
+        GitHub
+        </ExternalLink> for details)
```

Comment:
Doesn't this break on multiple lines now? I think a regular [GitHub](#) link is understandable enough